crates/bevy_gizmos/src/grid.rs

```diff
@@ -39,70 +101,54 @@ impl<T: GizmoConfigGroup> GridBuilder<'_, '_, '_, T> {
         self
     }
 
+    /// Toggle whether the outer edges of the grid along the x axis should be drawn.
+    /// By default, the outer edges will not be drawn.
+    pub fn outer_edges_x(mut self, outer_edges: bool) -> Self {
+        self.outer_edges[0] = outer_edges;
+        self
+    }
```

Comment:
I think we can do something like that, as the fields are not public and by default we have false in that field:
```suggestion
    pub fn outer_edges_x(mut self) -> Self {
        self.outer_edges[0] = true;
        self
    }
```